Evernote/Evernote.download.recipe

```diff
@@ -3,65 +3,64 @@
 <plist version="1.0">
 <dict>
     <key>Description</key>
-    <string>Downloads the current release version of Evernote
-
-Alternate Beta URL that can be used for SPARKLE_FEED_URL:
-http://update.evernote.com/prerelease/ENMac/EvernoteMacUpdate.xml
-</string>
+    <string>Downloads the latest verison of Evernote</string>
     <key>Identifier</key>
     <string>com.github.autopkg.download.Evernote</string>
     <key>Input</key>
-    <dict>
-        <key>NAME</key>
-        <string>Evernote</string>
-        <key>SPARKLE_FEED_URL</key>
-        <string>https://update.evernote.com/public/ENMacSMD/EvernoteMacUpdate.xml</string>
-    </dict>
-    <key>MinimumVersion</key>
-    <string>0.4.0</string>
-    <key>Process</key>
-    <array>
+	<dict>
+		<key>NAME</key>
+		<string>Evernote</string>
+		<key>SEARCH_URL</key>
+		<string>https://evernote.com/download</string>
+        <key>USER_AGENT</key>
+        <string>Mozilla/5.0 (Macintosh; Intel Mac OS X 11_0_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.193 Safari/537.36 Edg/86.0.622.68</string>
+	</dict>
+	<key>MinimumVersion</key>
+	<string>2.0.0</string>
+	<key>Process</key>
+	<array>
         <dict>
             <key>Processor</key>
-            <string>SparkleUpdateInfoProvider</string>
+            <string>URLTextSearcher</string>
             <key>Arguments</key>
             <dict>
-                <key>appcast_url</key>
-                <string>%SPARKLE_FEED_URL%</string>
+                <key>url</key>
+                <string>%SEARCH_URL%</string>
+                <key>request_headers</key>
+                <dict>
+                        <key>user-agent</key>
+                        <string>%USER_AGENT%</string>
+                </dict>
+                <key>re_pattern</key>
+                <string>(?P&lt;url&gt;https:\/\/cdn1\.evernote\.com\/boron\/mac\/builds\/Evernote-[\d.]+-[\d]+\.dmg)</string>
```

Comment:
This seems fragile - if they ever change the CDN base, this will break. Can you just look for the Evernote-*.dmg name and get the correct one?

Reply:
Would you prefer something a little more broad like this?

https:\/\/.*?\/Evernote-[\d.]+-[\d]+\.dmg